File: IOT/Nivedita-R_NiveditaRai_2125it1079_2/GettingStarted/GradingSystem.cpp
#include <iostream>

using namespace std;

int main()
{
    int a;
    cin>>a;
    if(a>90)
    cout<<"excellent"<<endl;
    else if(a<=90 && a>80)
    cout<<"good"<<endl;
    else if(a>70 && a<=80)
    cout<<"fair"<<endl;
    else if(a>60 && a<=70)
    cout<<"meets expectations"<<endl;
    else
    cout<<"below par"<<endl;
    

    return 0;
}

File: Web/Tushar-sahani_TusharSahani_2125cs1072_2/GettingStarted/GradingSystem.java
package coderspear;

import java.util.*;

public class GradingSystem {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int marks = sc.nextInt();

        System.out.println(marks > 90 ? "excellent"
                : marks > 80 && marks <= 90 ? "good"
                        : marks > 70 && marks <= 80 ? "fair"
                                : marks > 60 && marks <= 70 ? "meets expectations"
                                        : marks <= 60 ? "below par" : 0);
        sc.close();
    }
}